lldb/include/lldb/Utility/UUID.h

```diff
@@ -88,6 +88,9 @@ class UUID {
   DecodeUUIDBytesFromString(llvm::StringRef str,
                             llvm::SmallVectorImpl<uint8_t> &uuid_bytes);
 
+  /// Generate a random UUID.
```

Comment:
s/Generate/Generates ? (to be consistent with the other comments)

Reply:
I updated the existing comments to be imperative and updated this one to say "Create" to match the comment for the constructors.